src/store/slices/node/actionsAsync.ts

```diff
@@ -603,7 +606,7 @@ export const createExtraReducers = (
       const { balance, channelId, peerId, status, type } = action.payload;
       // find channel if it already exists
       const channelIndex = state.channels?.[type].findIndex(
-        (channel) => channel.channelId === channelId
+        (channel: any) => channel.channelId === channelId
```

Comment:
is this :any needed? i would assume it i inferred from state.channels type

Reply:
Good point, changed